daskms/fsspec_store.py

```diff
@@ -125,9 +130,7 @@ def __getitem__(self, key):
         return self.map[key]
 
     def _extend_path(self, path=""):
-        return (f"{self.full_path}{self.fs.sep}{path}"
-                if self.full_path
-                else self.full_path)
+        return self.join([self.full_path, path])
```

Comment:
I think if `path` is Falsey (i.e. "") then `full_path` should be returned, otherwise this will result in a file separator being added to `full_path`.

Reply:
Changed, but please check that I did it appropriately.